src/status_im/ui/screens/chat/views.cljs

```diff
@@ -320,7 +329,8 @@
         pan-responder (accessory/create-pan-responder position-y pan-state)
         space-keeper (get-space-keeper-ios bottom-space panel-space active-panel text-input-ref)
         set-active-panel (get-set-active-panel active-panel)
-        on-close #(set-active-panel nil)]
+        on-close #(set-active-panel nil)
+        messages (re-frame/subscribe [:chats/chat-messages-stream curr-chat-id])]
```

Comment:
it's better to move it inside messages-view , and use in condition, because we don't want to re-render other parts of chat view when messages are changed